examples/match_statements/src/main.sw

```diff
@@ -1,45 +1,48 @@
 script;
 
-fn foo() {}
-    // do something
-fn bar() {}
-
-    // do something
-enum SomeEnum {
-    A: u64,
-    B: bool,
-    C: b256,
+// helper functions for our example
+fn on_even(num: u64) {
+    // do something with even numbers
+}
+fn on_odd(num: u64) {
+    // do something with odd numbers
 }
 
-fn main() -> u64 {
-    let x = 5;
-
-    // Match as an expression.
-    let a = match 8 {
-        7 => 4,
-        9 => 5,
-        8 => 6,
-        _ => 100,
+fn main(num: u64) -> u64 {
+// Match as an expression
```

Comment:
Should these comments be indented to match the current block? ie:
```rust
// this:
fn main(num: u64) -> u64 {
    // Match as an expression
    let isEven = match num % 2 {
        0 => true,
        _ => false,
    };
}

// as opposed to this:
fn main(num: u64) -> u64 {
// Match as an expression
    let isEven = match num % 2 {
        0 => true,
        _ => false,
    };
}
```

Reply:
fixed, thank you!